java/ql/test/experimental/query-tests/security/CWE-089/src/main/MybatisSqlInjectionService.java

```diff
@@ -51,6 +52,17 @@ public void bad9(String name) {
 		sqlInjectionMapper.bad9(hashMap);
 	}
 
+	public String bad10(String name) {
+		// TODO: make useful tests
```

Comment:
How should proper tests for these sinks be like? Would a couple of real world examples (select, update, insert and delete) be enough?

Reply:
Ideally, you'd have (at least) a test line for each new sink being added.